src/cli/flags/groups.go

```diff
@@ -0,0 +1,23 @@
+package flags
```

Comment:
Since I was already halfway through filling out some boilerplate before I remembered to check what you'd added in this PR, I went ahead and stole this code to turn it into a boilerplate PR as well: https://github.com/alcionai/corso/pull/4057